schema.sql

```diff
@@ -1,5 +1,15 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+-- Database: vet_clinic
```

Comment:
- [ ] I do not see you creating the `animals table` in this file. You cannot insert data or run queries on a table that does not exist. Please create the `animals table`